dev/com.ibm.ws.microprofile.config.1.1_fat_tck/fat/src/com/ibm/ws/microprofile/config/tck/ConfigTCKLauncher.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2017,2022 IBM Corporation and others.
```

Comment:
strictly there should be space between the two
```suggestion
 * Copyright (c) 2017, 2022 IBM Corporation and others.
```